apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -412,6 +412,13 @@ export class NotificationService {
       existingCase.additionToConclusion,
     )
 
+    await this.sendEmail(
+      'Gæsluvarðhaldsfangelsi',
+      existingCase.prosecutor.email,
+      subject,
+      html,
+    )
```

Comment:
The first argument to sendEmail is the recipient's name, i.e. in this case it should be the name of the prosecutor. The '?' is to protect agains null pointer errors. By the time this email is sent, the prosecutor should have been set, but we have sometimes seen null pointers in the past. I would rather get an error from the email server than getting an unhandled null pointer.

```suggestion
    await this.sendEmail(
      existingCase.prosecutor?.name,
      existingCase.prosecutor?.email,
      subject,
      html,
    )
```